### PR TITLE
Feature: introduced converter properties

### LINKED ISSUE #85: Make 'camunda-7-adapter' configurable
### Is your feature request related to a problem? Please describe.

In the moment all Delegates are mapped to camunda-7-adapter.

It is a potential bottleneck if I would like to use my own adapter framework.

### Describe the solution you'd like

Make the target job type for delegates configurable.

Keep the value `camunda-7-adapter` as long as no other setting is provided.

### Describe alternatives you've considered

Nothing

### Additional context

No context

### PR DESCRIPTION
**Pull request by jonathanlukas**
## Description
Converter properties were introduced and now also replace NamespaceUri.

Until now, all properties like zeebe headers, namespaces or static attributes were hard-coded. Now, there is a properties file that consists of all these values.

## Additional context
This change originates in the requirement of customizing the adapter job type.

Closes #85 

## Testing your changes
There is a new test ConverterPropertiesTest that loads the properties and checks for some of them whether they are set or not.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an existing open issue)
- [x] New feature (non-breaking change which adds functionality to an extension)
- [ ] Breaking change (fix or feature that would cause existing functionality of an extension to change)
- [ ] Documentation update (changes made to an existing piece of documentation)

## Checklist:
- [x] My code adheres to the syntax used by this extension.
- [x] My pull request requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **Camunda Community Hub** documentation.
- [x] I have read the **Pull Request Process** documentation.
- [x] I have added or suggested tests to cover my changes suggested in this pull request.
- [x] All new and existing CI/CD tests passed.
- [x] I will /assign myself this issue, and add the relevant [issue labels] to it if they are not automatically generated by Probot.
- [x] I will tag @camunda-community-hub/devrel in a new comment on this issue if 30 days have passed since my pull request was opened and I have not received a response from the extension's maintainer.

### CODE REVIEW

File: backend-diagram-converter/core/src/main/resources/converter-properties.properties [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+## Camunda
+namespace.camunda.name=camunda
+namespace.camunda.uri=http://camunda.org/schema/1.0/bpmn
+## BPMN
+namespace.bpmn.name=bpmn
+namespace.bpmn.uri=http://www.omg.org/spec/BPMN/20100524/MODEL
+## XSI
+namespace.xsi.name=xsi
+namespace.xsi.uri=http://www.w3.org/2001/XMLSchema-instance
+# Static Attributes
```
Comment (ingorichtsmeier) [inline backend-diagram-converter/core/src/main/resources/converter-properties.properties:36]: Are these namespaces relevant to configure them? 

Access in the code becomes a lot more effort:
```
context.getProperties().getCamundaNamespace().getUri()
```
before:
```
NamespaceUri.CAMUNDA
```

**Review by ingorichtsmeier — CHANGES_REQUESTED** [review]
Generally, the externalization goes into the right direction. 

But I think that the properties for the namespaces are overengineered. Text constants should be sufficient for them. It is very tedious to access them now:

```
NamespaceUri.CAMUNDA -> context.getProperties().getCamundaNamespace().getUri()
```

File: backend-diagram-converter/core/src/main/resources/converter-properties.properties (hunk 1 above)
Comment (jonathanlukas) [reply backend-diagram-converter/core/src/main/resources/converter-properties.properties]: Done

File: backend-diagram-converter/core/src/main/resources/converter-properties.properties [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+zeebe-job-type.adapter=camunda-7-adapter
+## Script
+zeebe-job-type.script=script
+# Zeebe Headers
+## Script
+zeebe-header.script=script
+## Result variable
+zeebe-header.result-variable=resultVariable
+## Adapter job type
+zeebe-header.adapter-job-type=camunda-7-adapter
```
[the remark this answers is not in the document]
Comment (jonathanlukas) [reply backend-diagram-converter/core/src/main/resources/converter-properties.properties:12]: One is a header, one is a job type. They happen to be equal, yet they have a different meaning.
